static/js/zamboni/buttons.js

```diff
@@ -274,6 +274,7 @@ var installButton = function() {
             warn(gettext('Not available for your platform'));
             $button.addClass('concealed');
             $button.first().css('display', 'inherit');
+            $button.closest('.item.addon').addClass('incompatible');
```

Comment:
How was the CSS keyed before? Is there not a way to change this without modifying the JS?